experimental/segmenter/Cargo.toml

```diff
@@ -24,18 +24,21 @@ include = [
     "README.md"
 ]
 
+[package.metadata.workspaces]
+independent = true
```

Comment:
I'm not really sure what are the implications of this line

Reply:
`cargo ws version` can be used to bump the versions. By default all crates are versioned together.

Reply:
Should we remove these `[package.metadata.workspaces]` lines now?

Reply:
I'd still suggest `cargo ws version` to bump version, it's really convenient.